deepecho/sequences.py

```diff
@@ -110,27 +112,29 @@ def _convert_to_dicts(segments, context_columns):
         if context_columns:
             context = segment[context_columns]
             if len(context.drop_duplicates()) > 1:
-                raise ValueError('Context columns are not constant within each segment.')
```

Comment:
Is there any way to allow single lines like this?

Reply:
Sure, increased line-length to 99
https://github.com/sdv-dev/DeepEcho/pull/94/commits/42f595f77ae29cc0ef7ccecd96a2adf48bac5ee9